Client/Teztez/Flows/Common/ActivitiesIntro/ViewModel/ActivitiesIntroViewModel.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ActivitiesIntroViewModel.swift
+//  Teztez
+//
+//  Created by Adlet on 4/23/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+enum ActivityIntroType {
+    case backward, blender, schulte, colorMatch
+}
+
+struct ActivitiesIntroViewModel {
+    private var type: ActivityIntroType
+
+    var colors: [CGColor] {
+        switch type {
+        case .blender:
+            return [UIColor.coralLight.cgColor, UIColor.coralDark.cgColor]
+        case .backward:
+            return [UIColor.violetLight.cgColor, UIColor.violetDark.cgColor]
+        case .colorMatch:
+            return [UIColor.greenLight.cgColor, UIColor.greenDark.cgColor]
+        case .schulte:
+            return [UIColor.orangeLigt.cgColor, UIColor.orangeDark.cgColor]
+        }
+    }
+
+    var activityTitle: String {
```

Comment:
Also in this particular case (and the ones below) - when you are naming variables prefixes are needed to add more **context** and clarity to what you are talking about.

In this case **activityTitle** gives reader a hint that _title_ belongs to _activity_. But actually it is redundant here, because your class name already provides necessary context: **ActivitityIntroViewModel** - we already understand that we are talking about **activity**. So it is enough just to write **title**